command-parser.js

```diff
@@ -30,6 +30,8 @@ const MESSAGE_COOLDOWN = 5 * 60 * 1000;
 
 const MAX_PARSE_RECURSION = 10;
 
+const VALID_COMMAND_TOKENS = ['/', '!'];
```

Comment:
The only hardcoded part that remains after this is ``!`` being used for broadcasts.
I guess a BROADCAST_TOKEN could be defined as well.